templates/shortcode-form-grid.php

```diff
@@ -377,25 +376,22 @@ static function($term)use($tag_text_color,$tag_bg_color){
                                         elseif ( 'percentage' === $goal_format ) :
 
                                             echo sprintf( /* translators: %s: percentage of the amount raised compared to the goal target */
-                                                __( '<div class="form-grid-raised__details">
+                                                __( '
                                                    <span class="amount">%s%%</span>
-                                                   <span class="goal"><span>of </span><span>100&#37;</span>
-                                         </div>', 'give' ),
+                                                   <span class="goal"><span>of </span><span>100&#37;</span>', 'give' ),
                                                 round( $progress )
                                             );
 
                                         elseif ( 'donation' === $goal_format ) :
 
                                             echo sprintf( /* translators: 1: total number of donations completed 2: total number of donations set as goal */
                                                 _n(
-                                                    '<div class="form-grid-raised__details">
+                                                    '
                                                            <span class="amount">%1$s</span>
-                                                           <span class="goal"><span>of</span> %2$s <span>Donation</span></span>
-                                                </div>',
-                                                    '<div class="form-grid-raised__details">
+                                                           <span class="goal"><span>of</span> %2$s <span>Donation</span></span>',
+                                                    '
                                                            <span class="amount">%1$s</span>
-                                                           <span class="goal"><span>of</span> %2$s <span>Donations</span></span>
-                                                </div>',
+                                                           <span class="goal"><span>of</span> %2$s <span>Donations</span></span>',
```

Comment:
I would think that only `Donation/Donations` should be translated here. The surrounding HTML just complicates the translation. Also, I'm sure that we have `Donation/Donations` translated elsewhere in the plugin, so we could re-use that value if we isolate what needs to be translated.

Reply:
@kjohnson I ended up refactoring the sprintfs inside the tags to make it easier to read in https://github.com/impress-org/givewp/pull/6589/commits/1664d81f0cf0a12b9def3dc3fd65d4f1f6a60c35

<img width="1595" alt="Screen Shot 2022-10-20 at 9 56 32 AM" src="https://user-images.githubusercontent.com/75056371/197012121-a959daae-264c-4121-994b-150e751f44ca.png">

Reply:
Perfect.